README.rst

```diff
@@ -13,7 +13,7 @@ Spinal Cord Toolbox
 
```

Comment:
One other small thing. I notice in the mailing list form it says:

![image](https://user-images.githubusercontent.com/16181459/215142149-2d43b221-0ff6-4b2d-8747-95e3ccf4550d.png)

I would suggest that we change this wording to remove the "note", and also replace the word "advertising". 

This is a subjective opinion, of course, but I think both of these might be a bit scary-sounding? 

(For the former, I think there's already an expectation of privacy? So, by saying that you're *NOT* doing <insert bad thing>, it puts the idea of <bad thing> into a person's head, and makes the statement seem suspicious. For the latter, the word "advertising" is pretty loaded these days, so I think it might be better to say "announcements" like we do on our website/readme.)

Reply:
Proposed:

![image](https://user-images.githubusercontent.com/16181459/215144343-d9f0d133-46e3-446a-b64c-04535d10c303.png)

Reply:
perfect! i see you changed it already -- this is perfect, i agree with you-- thanks!